app/modules/video-players/video-ads.js

```diff
@@ -201,14 +202,30 @@ function init(player, options, slotTargeting) {
 	});
 
 	player.on('adRequest', (event) => {
-		const vastParams = vastParser.parse(event.tag, {
-			imaAd: event.ima && event.ima.ad,
-		});
+		const timeStamp = Math.round(window.performance && window.performance.now && window.performance.timing &&
+			window.performance.timing.navigationStart ? window.performance.now() +
+			window.performance.timing.navigationStart : Date.now()),
+			vastParams = vastParser.parse(event.tag, {
+				imaAd: event.ima && event.ima.ad,
+			});
 		vastDebugger.setVastAttributes(videoContainer, 'success', vastParams);
```

Comment:
Could you fix this place please? We should use `setVastAttributesFromVastParams` instead of `setVastAttributes`
https://github.com/Wikia/ad-engine/blob/dev/src/video/vast-debugger.js#L12

Reply:
Fixed.